translationCoreApps/VerseCheck
# WIP: Fix #3366/tW crash on projects missing verses

#### This pull request addresses:

Issue: https://github.com/unfoldingWord-dev/translationCore/issues/3366

Fixes tW crash on projects missing verses.  Adds null verse protection for normalizeString() to prevent tW crashing.  Also consolidates duplicate normalizeString() definitions into one place.

#### How to test this pull request:

import online project from https://git.door43.org/168b7e635905dd89/uw-gal-hsl

Open tW and should see not see empty panel.

#### Note:  coverage is down because duplicated code was removed.



---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/versecheck/85)
